functions/webhooks/twitter/FlexChannelUpdate.protected.ts

```diff
@@ -66,11 +84,7 @@ export const handler = async (
       if (status === 'INACTIVE') {
         await timeout(1000); // set small timeout just in case some cleanup is still going on
 
-        const removed = await context
-          .getTwilioClient()
-          .sync.services(context.SYNC_SERVICE_SID)
-          .documents(from)
-          .remove();
+        const removed = await cleanupUserChannelMap(context, from);
 
         console.log(`INACTIVE channel triggered map removal for ${from}, removed ${removed}`);
 
```

Comment:
For the line below, I'm not sure what the value of `removed` is if it did remove a document, but the message if it failed to remove will be:
`INACTIVE channel triggered map removal for <whatever 'from' is>, removed false`
Is "removed false" the best message?  Is there a message that's more clear?